containers/scripts/crlite-generate.sh

```diff
@@ -43,13 +43,20 @@ ls -latS ${ID}/revoked | head
 echo "crlite-fullrun: disk usage"
 du -hc ${ID}
 
-${workflow}/1-generate_mlbf ${ID} \
+if [ "x${DoNotUpload}x" == "xx" ] ; then
+  echo "archiving crls"
+  tar -czf ${ID}/crls.tar.gz -C ${CRLS} .
+  echo "uploading source materials"
+  ${workflow}/1-upload_data_to_storage ${ID} \
```

Comment:
Should this be `1-upload_artifacts_to_storage`? Or is there a missing file or something?

Reply:
ah, yeah, hg <-> git context switch  mishap.